src/content/apis/entertainment/genius.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Genius
+description: Search Songs, Artists and Lyrics. Get Lyric information for any Artist or Song from Genius.com
+categories:
+  - entertainment
+features:
+  https: true
+url: https://docs.genius.com/
+logo: https://docs.genius.com/images/snarly_wordmark_hires.png

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.